CMakeLists.txt

```diff
@@ -4,19 +4,34 @@ PROJECT (tinygltf)
 
 SET(CMAKE_CXX_STANDARD 11)
 
-ADD_EXECUTABLE ( loader_example
-  loader_example.cc
-  )
+add_library(tinygltf INTERFACE)
+
+target_compile_definitions(tinygltf
+        INTERFACE -DTINYGLTF_IMPLEMENTATION
+        -DTINYGLTF_USE_RAPIDJSON
+        -DSTB_IMAGE_IMPLEMENTATION
+        -DSTB_IMAGE_WRITE_IMPLEMENTATION)
+
+target_include_directories(tinygltf
+        INTERFACE $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+        ${RapidJSON_INCLUDE_DIRS})
+
+add_executable(loader_example
+        loader_example.cc
+        )
+
+target_link_libraries(loader_example
```

Comment:
I thought we were dropping the examples?

Reply:
I dropped the folder of examples as it had a bunch of glew dependencies that I didn't want to deal with. This one I made work - could drop it though, don't really need it

Reply:
I'm suprised there isnt a cmake option for this. Lets drop the example since it will pollute our build tree

Reply:
yeah I'm a fan of dropping the examples. After that its a LGTM